app/components/single-set.js

```diff
@@ -3,13 +3,7 @@ import { action } from '@ember/object';
 
 export default class SingleSetComponent extends Component {
   get isEnabled() {
-    const { set } = this.args;
-
-    if (set.winnerIndex != undefined) {
-      return false;
-    } else {
-      return true;
-    }
+    return !Boolean(this.args.set.winnerIndex != undefined);
```

Comment:
`!Boolean(this.args.set.winnerIndex)` isn't enough?

Reply:
No because winnerIndex might be equal 0 in which case it would return false.

Reply:
ah ok :)